Library/Formula/scmpuff.rb

```diff
@@ -0,0 +1,29 @@
+class Scmpuff < Formula
+  desc "Adds numbered shortcuts for common git commands"
+  homepage "https://mroth.github.io/scmpuff/"
+  url "https://github.com/mroth/scmpuff/archive/v0.1.1.tar.gz"
+  sha256 "cec3c9df41acb1735f2e8c1c9840d0481af0d996690f5a19a0a8fc4f06f97370"
+
+  depends_on "go" => :build
+
+  def install
+    # Go assumes things being compiled are located within a specific directory
```

Comment:
We can kill the notes. We do this in enough Go formulae that the process is apparently enough.